Внесены изменения под ТЗ9:

Vladyak1 opened the pull request:
- разработан API под методы TaskManager;
- написаны тесты под методы API.

Vladyak1 committed:
Внесены изменения под ТЗ9:
- разработан API под методы TaskManager;
- написаны тесты под методы API.

VladimirTitov4 commented on src/http/Tests/HistoryHandlerTest.java:
```diff
@@ -0,0 +1,21 @@
+package http.Tests;
```
пакеты пишутся с маленькой буквы tests

VladimirTitov4 commented on src/Tests/SubTaskTest.java:
```diff
@@ -0,0 +1,15 @@
+class SubTaskTest {
+
+    private Managers managers = new Managers();
+    SubTask subTask = new SubTask("Подзадача 1", "Описание подзадачи 1", "01.03.2024 02:00", 45);
```
Кажется, здесь ничего не тестируется

VladimirTitov4 commented on src/http/HttpTaskServer.java:
```diff
@@ -0,0 +1,42 @@
+        TaskManager manager = Managers.loadFromFile(new File("src\\service\\data.csv"));
+        startHttpTaskServer(manager);
+        try {
+            Thread.sleep(100000);
```
Для чего программа засыпает ?

Vladyak1 committed:
Новые изменения для ТЗ-9:
- файлы перемещены по папкам: тесты в src/test, остальное в src/main.

Vladyak1 committed:
Новые изменения для ТЗ-9:
- сервер больше не засыпает, а останавливается спустя время;
- тесты возвращены в SubtaskTest.
